addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js

```diff
@@ -65,7 +65,7 @@ const PublicLivechatWindow = Widget.extend({
         if (this._thread && this._thread._type === 'document_thread') {
             options.displayDocumentLinks = false;
         }
-        this._publicLivechatView = new PublicLivechatView(this, options);
+        this._publicLivechatView = new PublicLivechatView(this, this.messaging, options);
```

Comment:
from where does `this.messaging` come from?

Reply:
I forgot to put commit `[[IMP] im_livechat: have messaging in legacy PublicLivechatWindow](https://github.com/odoo/odoo/pull/95106/commits/375d1d5a9194167d6672f01c7ee5793951d4afab)` as requirement, which is what makes legacy `PublicLivechatWindow` have `this.messaging`.